### PR TITLE
Feat/spec test compatible

### PR DESCRIPTION
**Pull request by mouseless-eth**

### CODE REVIEW

File: src/rpc/rpcHandler.ts [hunk 1]
```diff
@@ -592,31 +595,39 @@ export class RpcHandler implements IRpcEndpoint {
...
-        } catch {
-            return null
-        }
 
-        if (op === undefined) {
+            if (foundOp === undefined) {
+                return null
+            }
+
+            if (slice(tx.input, 0, 4) === "0x765e827f") {
```
Comment (plusminushalf) [inline src/rpc/rpcHandler.ts:620]: What is this selector 0x765e827f? Can we create a constant and name it?

File: src/rpc/validation/SafeValidator.ts [hunk 2]
```diff
@@ -519,283 +494,131 @@ export class SafeValidator
...
+                to: this.entryPointSimulationsAddress,
                 data: callData
             },
             {
                 tracer: bundlerCollectorTracer
             }
         )
 
-        const lastResult = tracerResult.calls.slice(-1)[0]
-        if (lastResult.type !== "REVERT") {
```
Comment (plusminushalf) [inline src/rpc/validation/SafeValidator.ts:548]: Hey just to be sure we don't need this because we don't revert in the PimlicoSimulationsContract?

Comment (mouseless-eth) [reply src/rpc/validation/SafeValidator.ts:548]: good catch, re-added this check

File: src/rpc/rpcHandler.ts (hunk 1 above)
Comment (mouseless-eth) [reply src/rpc/rpcHandler.ts]: Good point, this magic constant/value is now removed

**Pull request merged**